packages/ffe-searchable-dropdown-react/src/MultiSelect/SearchableDropdownMultiSelect.stories.tsx

```diff
@@ -0,0 +1,241 @@
+    component: SearchableDropdownMultiSelect,
+    argTypes: {
+        postListElement: {
+            options: ['html', 'text', 'none'],
+            mapping: {
+                html: <span>Some text describing the list</span>,
+                text: 'Some text describing the list',
+                none: undefined,
+            },
+        },
+        optionBody: {
+            options: ['custom', 'none'],
+            mapping: {
+                custom: CustomOptionBody,
+                none: undefined,
+            },
+        },
+    },
+};
+export default meta;
+
+type Story = StoryObj<typeof SearchableDropdownMultiSelect>;
+
+export const Standard: Story = {
+    args: {
+        id: 'id',
+        labelledById: 'labelled-by-id',
+        dropdownList: fruits,
+        dropdownAttributes: ['displayName'],
+        searchAttributes: ['displayName', 'color'],
+        noMatch: { text: 'Søket ga ingen treff på frukt' },
+        inputProps: { placeholder: 'Søk' },
+        postListElement: 'none',
+    },
+    render: function Render({ id, labelledById, ...args }) {
+        return (
+            <>
+                <InputGroup
+                    label="Velg frukt"
+                    labelId={labelledById}
+                    inputId={id}
+                >
+                    <SearchableDropdownMultiSelect id="id" {...args} />
+                </InputGroup>
+            </>
+        );
+    },
+};
+
+export const MultipleDataResults: Story = {
+    args: {
+        ...Standard.args,
+        dropdownAttributes: ['displayName', 'color', 'amount'],
+    },
+    render: function Render({ id, labelledById, ...args }) {
+        return (
+            <InputGroup label="Velg frukt" labelId={labelledById} inputId={id}>
+                <SearchableDropdownMultiSelect
+                    id={id}
+                    labelledById={labelledById}
+                    {...args}
+                />
+            </InputGroup>
+        );
+    },
+};
+
+export const ControlledState: Story = {
+    args: {
+        ...Standard.args,
+        dropdownAttributes: ['displayName', 'color', 'amount'],
+    },
+    render: function Render({ id, labelledById, ...args }) {
+        const [selectedOptions, setSelectedOptions] = useState([
+            fruits[0],
+        ] as Record<string, any>[]);
+
+        const handleDropdownChange = (
+            item: Record<string, any>,
+            actionType: 'selected' | 'removed',
+        ) => {
+            const newList =
+                actionType === 'removed'
+                    ? selectedOptions?.filter(
+                          selectedItem => !isEqual(selectedItem, item),
+                      )
+                    : [...selectedOptions, item];
+            setSelectedOptions(newList);
+        };
+
+        return (
+            <InputGroup label="Velg frukt" labelId={labelledById} inputId={id}>
+                <SearchableDropdownMultiSelect
```

Comment:
<SearchableDropdownMultiSelect
                    id={id}
                    labelledById={labelledById}
           
                    selectedItems={selectedOptions}
                    onChange={handleDropdownChange}
                       {...args}
                />
                
                Slik att man skriver over med det man sender in

Reply:
Men onChange er definert i args også, da mister man jo poenget med storyen? 
![image](https://github.com/user-attachments/assets/12e5ea06-9f06-4d5e-9cdf-35bf615d8fbb)

Reply:
ja ok ... .jag tenker at lite av poenget også er att kunna endre ting i stories men jag er ikke noen expert på storybook. Her fratar man iaf muligheterna til endring.